src/js/components/user/githubUser.tsx

```diff
@@ -0,0 +1,263 @@
+import Avatar from '@salesforce/design-system-react/components/avatar';
+import Button from '@salesforce/design-system-react/components/button';
+import Card from '@salesforce/design-system-react/components/card';
+import DataTable from '@salesforce/design-system-react/components/data-table';
+import DataTableCell from '@salesforce/design-system-react/components/data-table/cell';
+import DataTableColumn from '@salesforce/design-system-react/components/data-table/column';
+import Modal from '@salesforce/design-system-react/components/modal';
+import classNames from 'classnames';
+import i18n from 'i18next';
+import React, { useEffect, useState } from 'react';
+
+import { GitHubUser } from '@/store/user/reducer';
+
+interface TableCellProps {
+  [key: string]: any;
+  item?: GitHubUser;
+}
+
+export const GitHubUserAvatar = ({ user }: { user: GitHubUser }) => (
+  <Avatar
+    imgAlt={user.login}
+    imgSrc={user.avatar_url}
+    title={user.login}
+    size="small"
+  />
+);
+
+export const UserCard = ({
+  user,
+  removeUser,
+  className,
+}: {
+  user: GitHubUser;
+  removeUser?: () => void;
+  className?: string;
+}) => (
+  <Card
+    className={classNames(className, 'collaborator-card')}
+    bodyClassName="slds-card__body_inner"
+    icon={<GitHubUserAvatar user={user} />}
+    heading={user.login}
+    headerActions={
+      removeUser && (
+        <Button
+          assistiveText={{ icon: i18n.t('Remove') }}
+          iconCategory="utility"
+          iconName="close"
+          iconSize="small"
+          iconVariant="border-filled"
+          variant="icon"
+          title={i18n.t('Remove')}
+          onClick={removeUser}
+        />
+      )
+    }
+  />
+);
+
+export const UserCards = ({
+  users,
+  removeUser,
+}: {
+  users: GitHubUser[];
+  removeUser: (user: GitHubUser) => void;
+}) => (
+  <div className="slds-grid slds-wrap slds-grid_pull-padded-xx-small">
+    {users.map((user) => {
+      const doRemoveUser = () => removeUser(user);
+      return (
+        <div
+          key={user.id}
+          className="slds-size_1-of-1
+            slds-large-size_1-of-2
+            slds-p-around_xx-small"
+        >
+          <UserCard user={user} removeUser={doRemoveUser} />
+        </div>
+      );
+    })}
+  </div>
+);
+
+const UserTableCell = ({ item, ...props }: TableCellProps) => {
+  /* istanbul ignore if */
+  if (!item) {
+    return null;
+  }
+  const { login } = item;
+  return (
+    <DataTableCell {...props} title={login} className="collaborator-grid">
```

Comment:
Is it possible to use this avatar and username as a target to toggle the checkbox?

Reply:
@stacyk Yes -- as part of that I brought this styling in line with the other modal a bit more. Feel free to review and cleanup, or let me know if there are minor changes you'd like (e.g. remove the outline on hover?).

![Screenshot 2020-01-30 17 46 40](https://user-images.githubusercontent.com/552316/73497303-7bcf0700-4388-11ea-9df8-82c7dfec80d3.png)

Reply:
@jgerigmeyer I removed outline on hover, removed focus on button, added table row hover (it feels more like a hover state, but I know the people working on this will know it is just a table style). I'd like to add cursor to pointer on checkbox but I know we decided not to go down that path. I don't like the double tabbing, but I know I asked for the name to be a clickable label, and because this is in a table we can't wrap the label around the checkbox and text. What do you think of the changes I just made?